### PR TITLE
Add task Html5-form

### PR DESCRIPTION
**Pull request by bugagashinka**
### HTML&CSS: Form layout

[Demo](https://bugagashinka.github.io/html5-form/) |
[Code base](https://github.com/bugagashinka/html5-form)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/bugagashinka/html5-form/index.html [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+    <link rel="stylesheet" href="css/index.css" />
+    <title>HTML5: Forms task</title>
+  </head>
+  <body>
+    <form action="#" method="POST">
+      <header>
+        <div class="container">
+          <section class="row heading">
+            <button type="button" class="button">Go back</button>
+            <svg class="logo">
```
Comment (yevhenorlov) [inline submissions/bugagashinka/html5-form/index.html]: Traditionally the logo in header and/or footer leads to the homepage. This should probably be wrapped in an `<a>` tag

File: submissions/bugagashinka/html5-form/index.html [hunk 2]
```diff
@@ -0,0 +1,519 @@
...
+              <div class="field-block">
+                <label for="expiry-date" class="input-label">Expiry date</label>
+                <input
+                  type="text"
+                  id="expiry-date"
+                  class="input-field input-date"
+                  name="expiry-date"
+                  title="Expiration date of your card DD/MM/YY"
+                  inputmode="numeric"
+                  pattern="([0-9]{2}[/]?){3}"
```
Comment (yevhenorlov) [inline submissions/bugagashinka/html5-form/index.html]: 99/99/99 shouldn't be a valid date
you may use `<input type="date">` if it's easier

File: submissions/bugagashinka/html5-form/index.html [hunk 3]
```diff
@@ -0,0 +1,519 @@
...
+              </div>
+            </div>
+          </section>
+        </div>
+      </main>
+    </form>
+    <footer>
+      <address class="container">
+        <div class="goal">
+          <h4><span class="icon">icon</span>What we do</h4>
```
Comment (yevhenorlov) [inline submissions/bugagashinka/html5-form/index.html]: placeholder for icon? 🙂

File: submissions/bugagashinka/html5-form/index.html [hunk 4]
```diff
@@ -0,0 +1,519 @@
...
+              <a href="tel:+971(0)44278128">Phone: +971 (0) 44278128</a>
+            </li>
+          </ul>
+        </div>
+        <div class="social-links">
+          <h4><span class="icon">icon</span>Connect with us</h4>
+          <ul class="list">
+            <li>Facebook</li>
+            <li>Twitter</li>
+            <li>Linkedin</li>
```
Comment (yevhenorlov) [inline submissions/bugagashinka/html5-form/index.html]: these should be links

File: submissions/bugagashinka/html5-form/index.html (hunk 2 above)
Comment (bugagashinka) [reply submissions/bugagashinka/html5-form/index.html]: Fix

File: submissions/bugagashinka/html5-form/index.html (hunk 1 above)
Comment (bugagashinka) [reply submissions/bugagashinka/html5-form/index.html]: Thanks for comments and recommendations

**Pull request merged**